recipes/modern-cpp-kafka/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
```

Comment:
Let me know if the message from the crawler is good enough to fix the error shown here.

Reply:
I have to use `# pylint: skip-file` for the v1 test, otherwise it shows `tools.cross_building` could not be found.